### PR TITLE
feat(landmarks): added

### LINKED ISSUE #16: Update atleast 5 landmarks for every city present in `cities.json`
There should be atleast 5 landmarks for a specified city.

### PR DESCRIPTION
**Pull request by dyaramatreya**
## What kind of change does this PR introduce?

- This PR updates the json file to make sure every city has at least 5 landmarks.
- For each of the existing city the landmarks have been added

- [ ] Bug Fix
- [ ] Documentation change
- [ ] Added a new city
- [x] Other(s)

## Why was this change needed?

Closes #16

### CODE REVIEW

**Comment by dyaramatreya** [comment]
@thenishantsapkota Please review the above changes.

File: data/borderlines.json [hunk 1]
```diff
@@ -197,6 +212,21 @@
...
+          "longitude": 26.6651,
+          "latitude": 87.6200
+        },
+        {
+          "name": "Buddhasubba Temple",
+          "longitude": 26.82193,
+          "latitude": 87.29652
+        },
+        {
+          "name": "Kosi Barrage",
```
Comment (thenishantsapkota) [inline data/borderlines.json:227]: Koshi Barrage isn't in urlabari

File: data/borderlines.json [hunk 2]
```diff
@@ -197,6 +212,21 @@
...
           "latitude": 26.6431722,
           "longitude": 87.6181256
+        },
+        {
+          "name": "Urlabari Multiple Campus",
+          "longitude": 26.6651,
+          "latitude": 87.6200
+        },
+        {
+          "name": "Buddhasubba Temple",
```
Comment (thenishantsapkota) [inline data/borderlines.json:222]: Neither is budhasubba temple

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please add the landmarks properly and fact check it before adding them

File: data/borderlines.json (hunk 1 above)
Comment (dyaramatreya) [reply data/borderlines.json]: Hi @thenishantsapkota 
Sorry, I relied on specific websites earlier. I've modified it with Urlabari Chowk from Google Maps

File: data/borderlines.json (hunk 2 above)
Comment (dyaramatreya) [reply data/borderlines.json]: And this, I've modified to Bhandari Krishi Farm.

File: data/borderlines.json [hunk 3]
```diff
@@ -197,6 +212,21 @@
...
           "latitude": 26.6431722,
           "longitude": 87.6181256
+        },
+        {
+          "name": "Urlabari Multiple Campus",
+          "longitude": 26.6651,
+          "latitude": 87.62
+        },
+        {
+          "name": "Bhandari Krishi Farm",
```
Comment (thenishantsapkota) [inline data/borderlines.json:222]: This isn't a significant landmark for Urlabari.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/borderlines.json [hunk 4]
```diff
@@ -79,6 +84,11 @@
           "name": "Himalaya Mahalaxmi Tea Garden",
           "latitude": 26.6539691,
           "longitude": 87.603649
+        },
+        {
+          "name": "Sano Baraghare",
```
Comment (thenishantsapkota) [inline data/borderlines.json:89]: Sano baraghare is a place in Damak not a landmark

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/borderlines.json (hunk 3 above)
Comment (dyaramatreya) [reply data/borderlines.json]: Added Shree Saraswati Secondary School in place of Bhandari Krishi Farm

File: data/borderlines.json (hunk 4 above)
Comment (dyaramatreya) [reply data/borderlines.json]: My bad, added New Amda Hospital in it's place

File: data/borderlines.json [hunk 5]
```diff
@@ -197,6 +212,21 @@
...
           "latitude": 26.6431722,
           "longitude": 87.6181256
+        },
+        {
+          "name": "Urlabari Multiple Campus",
+          "longitude": 26.6651,
+          "latitude": 87.62
+        },
+        {
+          "name": "Shree Saraswati Secondary School",
```
Comment (thenishantsapkota) [inline data/borderlines.json:222]: This isn't in urlabari still.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/borderlines.json (hunk 5 above)
Comment (dyaramatreya) [reply data/borderlines.json:222]: Is it? On Google Maps, it shows this address.

> MHMP+7W, Urlabari 56600, Nepal

Comment (dyaramatreya) [reply data/borderlines.json:222]: Do you still want me to make a change? I'll add another landmark then

Comment (dyaramatreya) [reply data/borderlines.json]: > This isn't in urlabari still.

Changed it to Madan Bhandari Hospital & Trauma Center.

**Review by thenishantsapkota — APPROVED** [review]
Looks good to me!
Thank you for your contribution @dyaramatreya!

**Comment by dyaramatreya** [comment]
> Looks good to me! Thank you for your contribution @dyaramatreya!

Thanks @thenishantsapkota!

**Pull request merged**